src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public enum Color {
+    GREEN("green"),
+    BLUE("blue"),
+    YELLOW("yellow"),
+    PINK("pink"),
+    GREY("grey");
+    private final String value;
+
+    Color(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
```

Comment:
Good point. But you either can use `Color.name().toLowerCase()`, see next comment.

Reply:
Pay attention here, currently is not critical.

Reply:
Sorry, I thought that it was just a tip about the possibility of solving it by another way for future